### ISSUE TITLE
[Idea] Device donation listing

### ISSUE THREAD

**Issue by fossdot** [fossunited/fossunited#679]
### Issue description

Similar to job posting, we can have a device donation and request on fossunited page.

Device could be: laptop, keyboard, etc.

**Comment by gi-yt** [comment]
I think it will be better to use something like telegram for this, doesn't make sense to make foss united into another olx

**Comment by rahulporuri** [comment]
> I think it will be better to use something like telegram for this, doesn't make sense to make foss united into another olx

Yes and No. Let's not have the individual device donations or requests on the page - but let's have a page stating that if anyone is interested, please reach out to us, and we will connect you with organizations that are in need of second-hand/used devices.

The intention was to make this a way for companies to indirectly support the FOSS United/Indian FOSS community by donating old devices, if they aren't able to donate money directly.

We only have one organization that was interested in participating (that needed devices) - and it'll be good to find at least one more organisation who is willing to accept donations before making this a public page
